dev/io.openliberty.jakarta.connectors.2.0/bnd.bnd

```diff
@@ -21,8 +21,8 @@ Export-Package: \
 	jakarta.resource.spi.security,\
 	jakarta.resource.spi.work
 
-Include-Resource: \
-  @${repo;jakarta.resource:jakarta.resource-api;2.0.0;EXACT}!/META-INF/NOTICE
+-includeresource: \
+  @${repo;jakarta.resource:jakarta.resource-api;2.0.0;EXACT}!/!(META-INF/maven/*|module-info.class)
```

Comment:
```suggestion
  @${repo;jakarta.resource:jakarta.resource-api;2.0.0;EXACT}!/!(META-INF/maven/*|module-info.class|doc-files/*)
```

Reply:
Are you sure about this one? The only file in `doc-files` is `speclicense.html` and part of the aim of this PR was to _include_ such license files.

Reply:
META-INF/LICENSE is in the bundle now.  I figured that was sufficient.  I assumed the doc files was for just that doc.  I didn't think it belonged in the bundle, but I can live with it being there as well.

Reply:
Will make the suggested change.

Reply:
done